0.9.0/.buildinfo

```diff
@@ -1,4 +0,0 @@
-# Sphinx build info version 1
```

Comment:
should this file have been deleted (as is the case from following README) or just updated?

Reply:
I think it's okay either way, as long as the resulting page is rendered correctly. We can come back to this if it affects GitHub's page deployment.